smoke-tests/apps/TelemetryProcessors/src/smokeTest/resources/applicationinsights-non-string-span-attributes.json

```diff
@@ -0,0 +1,45 @@
+{
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  },
+  "sampling": {
+    "percentage": 100
+  },
+  "preview": {
+    "processors": [
+      {
+        "type": "attribute",
+        "include": {
+          "matchType": "strict",
```

Comment:
From my understanding, the Github issue is related to regex?  It my be worth having `"matchType": "regexp"` in the test?

Reply:
but based on his json, it should be strict.

Reply:
we don't support non string regex.